packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -127,6 +127,29 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
     return t;
   });
 
+  const splitURLPath = useMemo(() => location.pathname.split('/'), []);
+  const decodedString = useMemo(
+    () => decodeURIComponent(splitURLPath[3]),
+    [splitURLPath],
+  );
+
+  //redirects users to All Discussions if they try to access a topic in the url that doesn't exist
+  useEffect(() => {
+    if (
+      !isLoadingTopics &&
+      decodedString &&
+      splitURLPath[2] === 'discussions' &&
+      splitURLPath.length === 4
+    ) {
+      const validTopics = topics?.some(
```

Comment:
why you check topics here? The bug is about deleted thread, not topic.

Reply:
I don't know why this code was included in this PR, this is from a different PR I had made to redirect users if they input the incorrect url that was merged already. But I will now change it here to account for custom domains, as mentioned above.